integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/authorization/CredentialCreateTest.java

```diff
@@ -20,6 +20,8 @@
 
 public class CredentialCreateTest extends AbstractIntegrationTest {
 
+    private static final Actor ENVCREATORB = Actor.useRealUmsUser(AuthUserKeys.ENV_CREATOR_B);
```

Comment:
This in not consistent with `CreateDhWithDatahubCreator`, in the other test it was not Static final Actor. I'm not a fan of these Class level extractions.